packages/gguf/src/gguf.ts

```diff
@@ -82,29 +82,31 @@ class RangeView {
  * Note: A good article about binary data in JS: https://javascript.info/arraybuffer-binary-arrays
  */
 
-function readVersionedSize(view: DataView, byteOffset: number, version: Version, littleEndian: boolean): bigint {
+function readVersionedSize(view: DataView, byteOffset: number, version: Version, littleEndian: boolean): {value: bigint, offset: number} {
```

Comment:
```suggestion
function readVersionedSize(view: DataView, byteOffset: number, version: Version, littleEndian: boolean): { value: bigint, offset: number } {
```

you might need to lint (i launched the CI runs)

Reply:
WDYT @mishig25 of the variable names